test/integration/index.ts

```diff
@@ -0,0 +1,17 @@
+/* eslint-disable import/export -- Tells webpack which files exist. */
+export * from './integration'
```

Comment:
This file needs to be at the bottom for browser tests (it's a weird thing that needs to be resolved, but isn't a priority right now)